pkg/contracts/src/CVStrategy.sol

```diff
@@ -297,26 +300,33 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
         this.withdraw(_member);
     }
 
-    function increasePower(address _member, uint256 _amountToStake) external view returns (uint256) {
+    function increasePower(address _member, uint256 _amountToStake) external returns (uint256) {
         //requireMemberActivatedInStrategies
 
         uint256 pointsToIncrease = 0;
         if (pointSystem == StrategyStruct.PointSystem.Unlimited) {
             pointsToIncrease = increasePowerUnlimited(_amountToStake);
         } else if (pointSystem == StrategyStruct.PointSystem.Capped) {
             pointsToIncrease = increasePowerCapped(_member, _amountToStake);
+        } else if (pointSystem == StrategyStruct.PointSystem.Quadratic) {
+            pointsToIncrease = increasePowerQuadratic(_member, _amountToStake);
         }
+        totalPointsActivated = pointsToIncrease;
+        emit PowerIncreased(_member, _amountToStake, totalPointsActivated);
         return pointsToIncrease;
     }
 
-    function decreasePower(address _member, uint256 _amountToUnstake) external view returns (uint256) {
+    function decreasePower(address _member, uint256 _amountToUnstake) external returns (uint256) {
         //requireMemberActivatedInStrategies
 
         uint256 pointsToDecrease = 0;
         if (pointSystem == StrategyStruct.PointSystem.Unlimited || pointSystem == StrategyStruct.PointSystem.Capped) {
             pointsToDecrease = decreasePowerCappedUnlimited(_member, _amountToUnstake);
+        } else {
+            pointsToDecrease = decreasePowerQuadratic(_member, _amountToUnstake);
         }
-
+        totalPointsActivated -= pointsToDecrease;
+        emit PowerDecreased(_member, _amountToUnstake, totalPointsActivated);
```

Comment:
here the subtract its correct but also what should go to the event its pointToDecrease or change the variable name in the event to be totalPointsActivated.